packages/yew-macro/tests/html_macro/dyn-prop-fail.stderr

```diff
@@ -0,0 +1,33 @@
+error: expected a valid Rust identifier
```

Comment:
maybe that message could be improved because it wasn't obvious to me why this would fail. not blocking though

Reply:
This error message is from the way it previously worked: components expect only valid rust identifiers for their prop names. Otherwise it would not make sense to make a struct from dynamic names here, it would basically be deserialization of sorts.
However, the error comes from Yew itself, and can actually be more specific (like my previous sentence)
```suggestion
error: components expect valid Rust identifiers for their property names
```
https://github.com/yewstack/yew/blob/2e8573125566efa772b9a3a9ad4846643257b74b/packages/yew-macro/src/props/component.rs#L197

Reply:
That's better indeed

Reply:
Done in 6427346